docs/_automation/powershell/README.md

```diff
@@ -34,10 +34,22 @@ The FinOps toolkit PowerShell module is a collection of commands to automate and
 
 ## 📥 Install the module
 
+The FinOps toolkit module requires PowerShell 7, which is supported on all major operating systems but is not installed by default. For details, see [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell). Once PowerShell has been installed, run the following:
```

Comment:
@flanakin maybe we should add a note about PS Core 7 being installed by default on the Azure Cloud Shell, which should probably be the preferred prompt where to run the module.

Reply:
I'd be fine with that assuming we validate it can be installed first. Have you tested loading the FTK module in CloudShell? If not, we can add it later.

Reply:
```suggestion
The FinOps toolkit module requires PowerShell 7, which is built into [Azure Cloud Shell](https://portal.azure.com/#cloudshell) and supported on all major operating systems. 

Azure Cloud Shell comes with PowerShell 7 and Azure PowerShell pre-installed. If you are not using Azure Cloud Shell, you will need to [Install PowerShell](https://learn.microsoft.com/powershell/scripting/install/installing-powershell) first and then run the following commands to install Azure PowerShell:
```